mondrian/src/main/java/mondrian/xmla/impl/MondrianXmlaServlet.java

```diff
@@ -20,7 +20,7 @@
 import java.io.File;
 import java.net.MalformedURLException;
 import java.net.URL;
-import javax.servlet.*;
+import jakarta.servlet.*;
 
```

Comment:
Import the specific classes instead of *. I know it was already there before but, just to follow the convention